packages/hsl-shared-components/src/Icons/ArrowRight.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Svg,
+  Path,
+  G,
+} from 'react-primitives-svg';
+
+import PropTypes from 'prop-types';
+
+import { IS_NATIVE, svgSize } from '../utils';
+
+export default function Icon({ fill, height, width, ...rest }) {
+  const flipNative = IS_NATIVE ? {
```

Comment:
Does this work properly when someone is applying animations to svg?
I'd ask the designers for these flipped versions if necessary.